src/client.py

```diff
@@ -74,78 +88,99 @@ def authorize(self, item: Item, request: Request, collection_id: str) -> dict:
                         "name": identity.get("name"),
                         "email": identity.get("email"),
                         "identity_provider": identity.get("identity_provider"),
-                        "identity_provider_display_name": identity.get(
-                            "identity_provider_display_name"
-                        ),
+                        "identity_provider_display_name": identity.get("identity_provider_display_name"),
                         "last_authentication": identity.get("last_authentication"),
                     }
 
-        auth = {
-            "requester_data": {
-                "client_id": token_info.get("client_id"),
-                "iss": token_info.get("iss"),
-                "sub": token_info.get("sub"),
-            },
-            "auth_basis_data": {
-                "authorization_basis_type": "group",
-                "authorization_basis_service": "groups.globus.org",
-                "authorization_basis": authorized_identities,
```

Comment:
Looks like we're not really using authorized_identities for anything other than raising a 403. Do we need [code](https://github.com/ESGF/stac-transaction-api/pull/23/files#diff-7c0673d4fdd044b4b5060131e82678aceb9bd6a5d045b56f108aa61b56af6026R82-R93)?